### PR TITLE
Create a PR template

### PR DESCRIPTION
**Pull request by DellaBitta**
[replace this line]: # (Describe your changes in detail.)
***

[replace this line]: # (Describe your testing in detail.)
***

### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [ ] New feature. A non-breaking change which adds functionality.
- [ ] Other, such as a build process or documentation change.
***

#### Notes
- Bug fixes and feature changes require an update to the `Release Notes` section of `release_build_files/readme.md`.
- Read the contribution guidelines [CONTRIBUTING.md](https://github.com/firebase/firebase-cpp-sdk/blob/main/CONTRIBUTING.md).
- Changes to the public API require an internal API review. If you'd like to help us make Firebase APIs better, please propose your change in a feature request so that we can discuss it together.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Type of Change
+Place an `x` the applicable box:
+- [ ] Bug fix. Add the issue # below if applicable.
+- [ ] New feature. A non-breaking change which adds functionality.
+- [ ] Other, such as a build process or documentation change.
+***
+
+#### Notes
+- Bug fixes and feature changes require an update to the `Release Notes` section of `release_build_files/readme.md`.*
```
Comment (jonsimantov) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:21]: What's the * for?

**Review by jonsimantov — COMMENTED** [review]
Looks good except for one nit.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (DellaBitta) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:21]: It's for BYOBB.

Comment (DellaBitta) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Fixed.

**Review by jonsimantov — APPROVED** [review]

**Pull request merged**